addons/stock/product.py

```diff
@@ -193,18 +412,123 @@ def _search_product_quantity(self, cr, uid, obj, name, domain, context):
 
             if operator == '=':
                 operator = '=='
-
+            ################################""""""""""""
```

Comment:
the code below was used to make real test between the to ways to compute qty (odoo orm and sql).